hanalytics/hanalytics-android/src/main/kotlin/com/hedvig/android/hanalytics/AndroidHAnalyticsService.kt

```diff
@@ -45,12 +40,12 @@ internal class HAnalyticsServiceImpl(
       "properties" to event.properties,
       "graphql" to event.graphql,
     )
+    d { "Stelios: requestJsonObject:$requestJsonObject" }
```

Comment:
Used when debugging I suppose

Reply:
I do this way too often 🙈
Yes will remove!